media-video/handbrake/handbrake-1.0.7.recipe

```diff
@@ -5,8 +5,8 @@ Convert from many common multimedia file formats, including unprotected \
 DVD or BluRay sources to a handful of modern output file formats."
 HOMEPAGE="http://handbrake.fr/"
 LICENSE="GNU GPL v2"
-COPYRIGHT="2003-2016 HandBrake Team"
-REVISION="3"
+COPYRIGHT="2003-2018 HandBrake Team"
```

Comment:
Revert this. You havent changed the recipe version, so this should stay unchanged,

Reply:
btw there is a new version here (1.1.1) you should bump the version.